crates/yttrium/src/transaction/send/safe_test.rs

```diff
@@ -449,11 +450,40 @@ mod tests {
         Ok(user_operation_hash)
     }
 
-    #[tokio::test]
-    async fn test_send_transaction() -> eyre::Result<()> {
-        let rpc_url = Config::local().endpoints.rpc.base_url;
-        let rpc_url: reqwest::Url = rpc_url.parse()?;
-        let provider = ReqwestProvider::<Ethereum>::new_http(rpc_url);
+    async fn use_faucet(
+        provider: ReqwestProvider,
+        faucet: LocalSigner<SigningKey>,
+        amount: U256,
+        to: Address,
+    ) -> eyre::Result<()> {
+        if amount > U256::from(20) {
+            panic!("You probably don't need that much");
```

Comment:
Basic check which we can tune to make sure we don't use excessive amount of test ETH. It is not free, so we should use the minimum amount necessary.